### ISSUE TITLE
1:100 scale does not correct display in new placeholder

### ISSUE THREAD

**Issue by allanchaplin** [nittygritty-uk/Atvero-Issues#132]
![screenshot 2019-01-09 at 16 35 58](https://user-images.githubusercontent.com/53964/50913690-ca7fcb00-142c-11e9-9d0d-5b9fb019dd12.png)

**Comment by LBoyle** [comment]
I fixed this on LSB26 by re-entering the "1:100" option on the DynamicParts table in SP, which suggests something is up with provisioning. 
This is the culprit:
```
part.value == part.title ?
              part.title : part.value ++ ": " ++ part.title
```

**Comment by LBoyle** [comment]
Fixed in PR #180
